lms/static/js/student_account/views/HintedLoginView.js

```diff
@@ -23,9 +28,11 @@
             },
 
             render: function() {
-                $(this.el).html(_.template(this.tpl)({
+                var output = HtmlUtils.template(this.tpl)({
```

Comment:
I would lose this variable as it is only accessed once, and it will make it harder for the linter to see that the HTML is escaped.